packages/back-end/src/integrations/Databricks.ts

```diff
@@ -14,6 +15,17 @@ export default class Databricks extends SqlIntegration {
       encryptedParams
     );
   }
+  isWritingTablesSupported(): boolean {
+    return true;
+  }
+  dropUnitsTable(): boolean {
+    return true;
```

Comment:
super nit: do we want any permitions checks around  isWritingTablesSupported and dropUnitTable to make sure that nothing slips through the cracks

Reply:
I suggested in slack: if your org has pipeline mode on, to make it work for all users, they basically must have permission to write and drop if they have permission to `runQuery` or else pipeline mode will need to be enabled user-by-user.